src/types.ts

```diff
@@ -13,13 +13,23 @@ export interface CaptureResult {
 }
 export type CaptureCallback = (response: any, data: any) => void
 
+export type UsefulElements = 'a' | 'button' | 'form' | 'input' | 'select' | 'textarea' | 'label'
```

Comment:
`Useful` is super vague, how about `AutocaptureCompatibleElement`? This won't really ever be imported by users anyway, and a more explicit name will be clearer when reading the type

Reply:
good point, I based it on the usage of `usefulElements` renamed that too to `autocaptureCompatibleElements`